modules/samtools/view/main.nf

```diff
@@ -11,28 +11,29 @@ process SAMTOOLS_VIEW {
         mode: params.publish_dir_mode,
         saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
 
-    conda (params.enable_conda ? 'bioconda::samtools=1.13' : null)
+    conda (params.enable_conda ? "bioconda::samtools=1.14" : null)
     if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
-        container "https://depot.galaxyproject.org/singularity/samtools:1.13--h8c37831_0"
+        container "https://depot.galaxyproject.org/singularity/samtools:1.14--hb421002_0"
     } else {
-        container "quay.io/biocontainers/samtools:1.13--h8c37831_0"
+        container "quay.io/biocontainers/samtools:1.14--hb421002_0"
     }
 
     input:
     tuple val(meta), path(input)
     path fasta
 
     output:
-    tuple val(meta), path("*.bam") , optional: true, emit: bam
-    tuple val(meta), path("*.cram"), optional: true, emit: cram
-    path  "versions.yml"                           , emit: versions
+    tuple val(meta), path("*.bam") , emit: bam , optional: true
+    tuple val(meta), path("*.cram"), emit: cram, optional: true
```

Comment:
so `optional: true` in the end, I like that

Reply:
Thats what I saw harshil do in a bunch of PRs on monday.